packages/kbn-pm/src/utils/bazel/setup_remote_cache.ts

```diff
@@ -35,7 +35,10 @@ async function upToDate(settingsPath: string) {
 
 export async function setupRemoteCache(repoRootPath: string) {
   // The remote cache is only for Elastic employees working locally (CI cache settings are handled elsewhere)
-  if (process.env.CI || !(await isElasticCommitter())) {
+  if (
+    process.env.FORCE_BOOTSTRAP_REMOTE_CACHE !== 'true' &&
```

Comment:
Interesting side effect here gives third party devs an easy option to enable remote bazel caching without using a fake Elastic email, in case any of them are lurking about :)